### PR TITLE
Adding occupied spot number attribute

### PR DESCRIPTION
**Pull request by jpnbastos**
PR to close issue #15

### CODE REVIEW

**Comment by albertoabellagarcia** [comment]
Thank you very much for your PR. I am afraid that you have worked very hard in points that are not 'useful'. 
The only documents that you need to contributed to, are the examples and the schema.json. (And eventually the notes.yaml if you want to customize the spec, and of course the contributors.yaml at subject level)
 The rest of the documentation is generated automatically from them. 
So from your PR the only thing to be reviewed is the schema.json, and on the other hand you did not update the examples which are relevant this case.
See the contributor manual https://docs.google.com/presentation/d/e/2PACX-1vTs-Ng5dIAwkg91oTTUdt8ua7woBXhPnwavZ0FxgR8BsAI_Ek3C5q97Nd94HS8KhP-r_quD4H0fgyt3/pub?start=false&loop=false&delayms=3000#slide=id.p1 for further insights 
Actions on your part:
1) Please extend your PR to include the examples
2) 'drop' the modifications in the spec.md, spec-ES.md, model.yaml, etc  because they will be overwritten once approved the schema.json and the examples modifications. 
if you have any other doubt please check the manual or just drop me an email alberto.abella@fiware.org

**Comment by jpnbastos** [comment]
Oh I see, I indeed did not consider that possibility. I will do so :) 

Thank you for the info!

**Comment by jpnbastos** [comment]
Btw, the contributor manual. I was looking for such a document before but I could not. Of course it can just mean that I did not look well or it missed me, but I would suggest adding it to the README.md of this project and others. :)

**Comment by jpnbastos** [comment]
It is done. Let me know if anything else is needed 👍

File: OnStreetParking/examples/example-normalized-ld.jsonld.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
...
...urs__value__blueZonePermit_", "requiredPermit__type_", "requiredPermit__value__0_", "requiredPermit__value__1_", "allowedVehicleType__type_", "allowedVehicleType__value_", "chargeType__type_", "chargeType__value__0_", "refParkingGroup__type_", "refParkingGroup__object__0_", "refParkingGroup__object__1_", "totalSpotNumber__type_", "totalSpotNumber__value_", "location__type_", "location__value__type_", "location__value__coordinates__0__0__0_", "location__value__coordinates__0__0__1_", "location__value__coordinates__0__1__0_", "location__value__coordinates__0__1__1_", "location__value__coordinates__0__2__0_", "location__value__coordinates__0__2__1_", "location__value__coordinates__0__3__0_", "location__value__coordinates__0__3__1_", "location__value__coordinates__0__4__0_", "location__value__coordinates__0__4__1_", "areaServed__type_", "areaServed__value_", "maximumAllowedStay__type_", "maximumAllowedStay__value_", "extraSpotNumber__type_", "extraSpotNumber__value_", "availableSpotNumber__type_", "availableSpotNumber__value_", "availableSpotNumber__observedAt_", "occupiedSpotNumber__type_", "occupiedSpotNumber__value_", "occupiedSpotNumber__observedAt_", "@context__0_", "@context__1_"
```
Comment (albertoabellagarcia) [inline OnStreetParking/examples/example-normalized-ld.jsonld.csv]: this file is generated automatically so it is not necessary (either convenient to be in the PR)

File: OnStreetParking/examples/example-normalized.json.csv [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
...pe_", "category__value__0_", "category__value__1_", "category__value__2_", "permitActiveHours__value__blueZonePermit_", "requiredPermit__value__0_", "requiredPermit__value__1_", "allowedVehicleType__value_", "chargeType__value__0_", "refParkingGroup__type_", "refParkingGroup__value__0_", "refParkingGroup__value__1_", "totalSpotNumber__value_", "location__type_", "location__value__type_", "location__value__coordinates__0__0__0_", "location__value__coordinates__0__0__1_", "location__value__coordinates__0__1__0_", "location__value__coordinates__0__1__1_", "location__value__coordinates__0__2__0_", "location__value__coordinates__0__2__1_", "location__value__coordinates__0__3__0_", "location__value__coordinates__0__3__1_", "location__value__coordinates__0__4__0_", "location__value__coordinates__0__4__1_", "areaServed__value_", "maximumAllowedStay__value_", "dateModified__type_", "dateModified__value_", "extraSpotNumber__value_", "availableSpotNumber__value_", "availableSpotNumber__metadata__timestamp__value_", "availableSpotNumber__metadata__timestamp__type_", "occupiedSpotNumber__value_", "occupiedSpotNumber__metadata__timestamp__value_", "occupiedSpotNumber__metadata__timestamp__type_"
```
Comment (albertoabellagarcia) [inline OnStreetParking/examples/example-normalized.json.csv]: this file is generated automatically so it is not necessary (either convenient to be in the PR)

File: OnStreetParking/examples/example.json.csv [hunk 3]
```diff
@@ -1,2 +1,2 @@
...
\ No newline at end of file
+"id_", "type_", "category__0_", "category__1_", "category__2_", "allowedVehicleType_", "chargeType__0_", "requiredPermit__0_", "requiredPermit__1_", "permitActiveHours__blueZonePermit_", "maximumAllowedStay_", "availableSpotNumber_", "occupiedSpotNumber_", "totalSpotNumber_", "extraSpotNumber_", "dateModified_", "location__type_", "location__coordinates__0__0__0_", "location__coordinates__0__0__1_", "location__coordinates__0__1__0_", "location__coordinates__0__1__1_", "location__coordinates__0__2__0_", "location__coordinates__0__2__1_", "location__coordinates__0__3__0_", "location__coordinates__0__3__1_", "location__coordinates__0__4__0_", "location__coordinates__0__4__1_", "areaServed_", "refParkingGroup__0_", "refParkingGroup__1_"
```
Comment (albertoabellagarcia) [inline OnStreetParking/examples/example.json.csv]: this file is generated automatically so it is not necessary (either convenient to be in the PR)

**Review by albertoabellagarcia — CHANGES_REQUESTED** [review]
It's ok but you have to remove the .csv files from the PR

**Comment by jpnbastos** [comment]
they are now removed. thanks!

**Comment by jpnbastos** [comment]
Hi Alberto, 

Is more work needed on this PR?

**Comment by albertoabellagarcia** [comment]
All files are right but the schema.json. Sorry to say that you did not remember to include the description in the change in the json schema for the data model. Please see the actual version of [schema.json](https://github.com/smart-data-models/dataModel.Parking/blob/master/OffStreetParking/schema.json) and the [contribution manual](https://bit.ly_contriubution_manual), page 9. 
Any doubt, please let me know.

**Comment by jpnbastos** [comment]
No problem, I have now added the description of the attribute, but as well update the schema. I noticed that it was not the latest schema version. Please have a look.

**Comment by albertoabellagarcia** [comment]
nice, thanks for your contribution

**Pull request merged**